### ISSUE TITLE
Twitch integration not working

### ISSUE THREAD

**Issue by badoge** [Pinsplash/halflife2chaos#52]
**Describe the bug**
I've followed all the instructions in the readme but I couldn't get the Twitch integration to work, the script log shows the messages posted in chat but the text source in OBS is stuck on "Vote #-1" and typing commands (!A, !B, !C, !D) does nothing

The only errors I get are "[Unknown Script] poll unexpected exception: " after connecting with twitch
and 
"[Unknown Script] poll failed [WinError 64] The specified network name is no longer available" after I reload the script in OBS

I was thinking that it might have to do with windows firewall but the errors still persist after disabling it 

**To Reproduce**
No clear steps to mention, just setup the twitch integration 

**Expected behavior**
The text source should show the effects and chat commands that chatters can use but it just says "Vote #-1" 

**Additional context**
mod version 0.2.6 - windows 10 - OBS 29.1.1 - latest steam beta client if that matters

**Comment by badoge** [comment]
yes, I just used the default value for the script in OBS and for the launch options in steam other than the password

**Comment by badoge** [comment]
I'm just using the default launch options from the readme with my rcon password
The game doesn't show up on the listening ports list

**Comment by acuifex** [comment]
that means the game doesn't start rcon socket for some reason. are you absolutely sure there is `+developer 0 ` at the start? can you give me command line string for the game?

https://superuser.com/questions/415360/how-do-i-find-out-command-line-arguments-of-a-running-program

**Comment by acuifex** [comment]
i have no fucking idea then. i guess it's having issues seeing `-usercon`? maybe it's interpreting all options as a single blob?
`net_status` to see if it done at least anything and then try `net_start` again?

**Comment by badoge** [comment]
I get this error when running net_start
![image](https://github.com/Pinsplash/halflife2chaos/assets/18620902/5a07dc26-b956-4ee5-bc61-15596bfff6f5)

![image](https://github.com/Pinsplash/halflife2chaos/assets/18620902/469732f5-e6d2-41d5-933e-fc5e31b3cc61)

**Comment by acuifex** [comment]
https://stackoverflow.com/questions/17123380/c-windows-socket-error-10013

i can't really help you any more since i'm a linux user. google search keywords: `bind localhost WSAEACCES 10013`

**Comment by Unmoon** [comment]
`WSAEACCES` is caused by something else already using the port. For badoge seemed that it was some other software, but it might be worthwhile to document the [`-port`](https://developer.valvesoftware.com/wiki/Command_Line_Options) command line option, and how that can be given in the OBS settings.

**Comment by badoge** [comment]
I found the problem  ![image](https://github.com/Pinsplash/halflife2chaos/assets/18620902/475d5b2d-cdfc-45b2-bff5-ae424537935c)
some other process was using the same port 
![image](https://github.com/Pinsplash/halflife2chaos/assets/18620902/151c5307-fae8-4332-b546-f52cae277a1d)
killing the other process fixed it

Sorry for wasting your time
